ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js

```diff
@@ -25,6 +25,26 @@ const store = require('../../js/store.js')
 const QuerySettingsView = require('../query-settings/query-settings.view.js')
 const user = require('../singletons/user-instance.js')
 
+import query from '../../react-component/utils/query'
+
+const fetchFacets = memoize(async attr => {
+  const json = await query({
```

Comment:
What source is used if we don't specify one?

Reply:
The cql endpoint defaults to local source if none are specified

Reply:
Thanks for the confirmation!